### PR TITLE
Invite backup3

### PR DESCRIPTION
**Pull request by jerry-jma**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -29,6 +29,16 @@ export default function App() {
    * 3. Add the route path after localhost:3000
    *
    */
+  const [inviteMessage, setInviteMessage] = useState('');
```
Comment (kyle-sablan) [inline src/App.js:32]: Would this be the best spot for invitation form state data?

File: src/App.js [hunk 2]
```diff
@@ -44,15 +54,32 @@ export default function App() {
...
-          <Route path="invitation-form" element={<InvitationForm />} />
-          <Route path="rsvp-preview" element={<RSVP />} />
+          <Route
+            path="choose-content"
+            element={<ChooseContent setChosedTheme={setChosedTheme} />}
+          />
+          <Route
+            path="invitation-form"
+            element={
+              <InvitationForm
```
Comment (kyle-sablan) [inline src/App.js:68]: related to the above comment, but would we need to pass the data down from this level?

File: src/Invitation/InvitationForm.js [hunk 3]
```diff
@@ -59,20 +77,21 @@ export default function InvitationForm() {
     <Container
       style={{
         width: '65%',
-        // backgroundImage: 'url(http://tineye.com/images/widgets/mona.jpg)',
-        // backgroundRepeat: 'no-repeat',
-        // height: '100%', ----------TO BE REVISED--------------
+        background: `url(${chosedTheme}) no-repeat center center fixed`,
+        backgroundSize: 'cover',
+        // backgroundColor: '#EED5DE',
+        // backgroundRepeat: 'no-repeat', -------------To be revised
```
Comment (kyle-sablan) [inline src/Invitation/InvitationForm.js:83]: comments may be cleared out

**Review by kyle-sablan — COMMENTED** [review]
Good job on hooking up the multi item carousel! Left a few comments on some of the structure of the code.

File: src/Invitation/BrowseTheme.js [hunk 4]
```diff
@@ -2,37 +2,41 @@ import React from 'react';
...
-          </h5>
-          <h5>
-            <i className="fas fa-mouse-pointer" />
-            Write a personal message for your guests
-          </h5>
-          <Link to="/choose-content">
-            <BlueButton>browse themes</BlueButton>
+          <div className="mt-5">
+            <h5>
+              <i className="fas fa-gift" /> Customize your invitation with
```
Comment (DylanDKim) [inline src/Invitation/BrowseTheme.js:20]: make sure to use styled fonts! -lmk if you need more styles

File: src/Invitation/InvitationForm.js [hunk 5]
```diff
@@ -105,12 +128,13 @@ export default function InvitationForm() {
...
             multiple
           />
+          <div>💡Pro Tip: A vertical photo will look best.</div>
           <div className="d-flex justify-content-center mt-2">
             <PlumButton onClick={addPicToGallery}>Add more photos</PlumButton>
           </div>
         </Form.Group>
         <div className="d-flex justify-content-end">
-          <Link to="/rsvp-preview" state={{ from: galleryURL }}>
+          <Link to="/event/1/rsvp-preview">
```
Comment (DylanDKim) [inline src/Invitation/InvitationForm.js:137]: for final product tomorrow, try to use the useParams() id, lmk if you need help with it tmr morning

**Review by kyle-sablan — APPROVED** [review]
The earlier comments can be good for discussion later on but isnt imperative for the PR.

**Review by DylanDKim — APPROVED** [review]

**Comment by DylanDKim** [comment]
critical change - use of useParams() - see my notes in the slack channel for examples - but ill pull it

**Pull request merged**